crates/kitsune_p2p/fetch/src/respond.rs

```diff
@@ -54,20 +54,24 @@ impl<C: FetchResponseConfig> FetchResponseQueue<C> {
     }
 
     /// Enqueue an op to be sent to a remote.
-    pub fn enqueue_op(&self, space: KSpace, user: C::User, op: KOpData) {
+    pub fn enqueue_op(&self, space: KSpace, user: C::User, op: KOpData) -> bool {
```

Comment:
Return a value for testing to understand whether a value was going to be sent or not